server/server.js

```diff
@@ -60,6 +60,20 @@ const server = net
         case MessageTypes.move:
           break
         case MessageTypes.start:
+          const challenged = waitList.find(user => user.nickname === aux.payload)
+          const challenger = waitList.find(user => user.socket === socket)
+          if (challeged && challenger) {
+            let removed = waitList.splice(waitList.indexOf(challegend), 1)
+            playingList.push(removed[0])
+            removed = waitList.splice(waitList.indexOf(challenger), 1)
+            playingList.push(removed[0])
+
+            socket.write(MessageStructure.messageStart(MessageTypes.accepted, challenger))
+            challenged.socket.write(MessageStructure.asyncStart(MessageTypes.accepted, challenged))
```

Comment:
Aqui tá enviando o objeto challenger pro challenger, e o challenged pro challenged. Tem que inverter.
E também não acho que seja necessário enviar todo o objeto, só a string do oponente. Então, `challenger.nickname`.

Reply:
Acho que deve ir o objeto sim devido ao fato de ter que enviar o X e o O também

Reply:
É. Pode ser melhor enviar o objeto todo. Se o front-end for dizer pro cara qual o símbolo dele, no caso.